objects/avalicrafting/coloriser/avalicoloriser.object

```diff
@@ -0,0 +1,42 @@
+{
+  "objectName" : "avalicoloriser",
+  "colonyTags" : [ "avali", "avalicamp", "crafting" ],
+  "rarity" : "Common",
+  "printable" : true,
+  "description" : "A specialised, hybrid coloriser used for bulk coloring.",
+  "shortdescription" : "^orange;Avali Coloriser^white;",
+  "race" : "avali",
+  "category" : "crafting",
+  "price" : 250,
+
+  "apexDescription" : "Oh my! I absolutely must study this some time.",
```

Comment:
Maybe poke the racial descriptions a bit so it's not just a copy of the Nanolathe

Reply:
Fixed in 085bbdc